packages/account/src/Components/api-token/api-token-clipboard.tsx

```diff
@@ -25,16 +36,21 @@ const WarningDialogMessage = () => (
             <WarningNoteBullet message={localize('Create or delete API tokens for trading and withdrawals')} />
             <WarningNoteBullet message={localize('Modify account settings')} />
         </div>
-    </>
+    </React.Fragment>
 );
 
-const ApiTokenClipboard = ({ scopes, text_copy, info_message, success_message, popover_alignment = 'bottom' }) => {
+const ApiTokenClipboard = ({
+    scopes,
+    text_copy,
+    info_message,
+    success_message,
+    popover_alignment = 'bottom',
+}: TApiTokenClipboard) => {
     const [is_copied, setIsCopied] = React.useState(false);
     const [is_modal_open, setIsModalOpen] = React.useState(false);
     const [is_popover_open, setIsPopoverOpen] = React.useState(false);
     const isMounted = useIsMounted();
-    let timeout_clipboard = null;
-    let timeout_clipboard_2 = null;
+    let timeout_clipboard: NodeJS.Timeout | undefined, timeout_clipboard_2: NodeJS.Timeout | undefined;
```

Comment:
why we are deleting here default value 'null'?
I believe we should avoid using 'undefined ' as a type.

Reply:
They are not deleted. I just fixed the lint error

<img width="791" alt="Screen Shot 2022-11-02 at 12 07 06 PM" src="https://user-images.githubusercontent.com/98398322/199433929-d77f0e32-4614-459e-a20a-82c4cead3207.png">